docs/graphs.ipynb

```diff
@@ -293,6 +293,31 @@
         "%tensorboard --logdir logs"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9PFgFjlPEqXb"
+      },
+      "source": [
+        "## TensorBoard.dev\n",
```

Comment:
It seems too prominent to make this a `##` header, since then it's at the same level as the headers for Op-level Graph and Conceptual Graph.  The result is that the following content which is supposed to be about the Op-level graph actually shows up under the header for TensorBoard.dev instead:

![image](https://user-images.githubusercontent.com/710113/120709789-26ba6e00-c472-11eb-8aaf-1328ceb3110d.png)

I'd either skip the TensorBoard.dev header entirely (since it's not really a full "section" about TB.dev; it's just a single code example) or at least make it `###` or `####` so it doesn't disrupt the flow.

Reply:
Good call.  Removed the header line entirely.